html/components/autocomplete.stories.js

```diff
@@ -272,7 +272,7 @@ export const defaultInvalid = () => {
 
       <ul
         data-sprk-autocomplete="results"
-        class="sprk-c-Autocomplete__results sprk-u-Display--none"
+        class="sprk-c-Autocomplete__results sprk-c-Autocomplete__results--is-hidden"
```

Comment:
This is-hidden for autocomplete existed but wasnt in use. It looks like i had commented on the autocomplete PR to make the class and it was made but never put into place. I put it into place now.

Reply:
No problem!